xen/include/public/io/sndif.h

```diff
@@ -453,8 +491,10 @@
 #define XENSND_FIELD_FE_VERSION         "version"
 #define XENSND_FIELD_VCARD_SHORT_NAME   "short-name"
 #define XENSND_FIELD_VCARD_LONG_NAME    "long-name"
-#define XENSND_FIELD_RING_REF           "ring-ref"
-#define XENSND_FIELD_EVT_CHNL           "event-channel"
+#define XENSND_FIELD_REQ_RING_REF       "req-ring-ref"
+#define XENSND_FIELD_REQ_EVT_CHNL       "req-event-channel"
```

Comment:
Not so good idea to rename these macros. As it may be already used by some applications (libxl) and it will require to change it everywhere.

Reply:
done